src/main/scala/uk/gov/hmrc/githubclient/ExtendedRepository.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmrc.githubclient;
```

Comment:
Do we need a java class? If so, it should probably be under `src/main/java`, but I'd expect we can use a scala case class (as we do `ExtendedContentsService` with `GhCodeResult` etc.)

Reply:
@colin-lamed The reason I stuck with Java here is that the org.eclipse.egit.github.core library is using the Gson library for JSON object parsing, which is written for Java. From my reading is seemed that there can be issues using Gson with Scala classes, so I thought it made pragmatic sense to stick with Java for this model. However, I can try converting it to a Scala class and see if the JSON parsing works without wrangling.

Reply:
If we do have Java classes, they should go in a `src/main/java` directory. This one is under `src/main/scala`. Seems a bit odd.

Reply:
@cjwebb Agreed. However converting to Scala seems to work fine with Gson so will do that rather than moving.